dotcom-rendering/src/amp/components/Caption.tsx

```diff
@@ -3,8 +3,8 @@ import { css } from '@emotion/react';
 
 import { text, textSans } from '@guardian/source-foundations';
 
-import { pillarPalette_DO_NOT_USE } from '@root/src/lib/pillars';
-import TriangleIcon from '@frontend/static/icons/triangle.svg';
+import { pillarPalette_DO_NOT_USE } from '../../lib/pillars';
```

Comment:
👀

Reply:
Could also be marked as `@deprecated`… but it won’t prevent peeps from using it 😉